srcjs/output_binding_image.js

```diff
@@ -111,6 +111,12 @@ $.extend(imageOutputBinding, {
       };
     }
 
+    // Set optional alt-attribute
+    var alttext = $el.attr("alt");
+    if (alttext && alttext !== "") {
+      img.setAttribute("alt", alttext);
```

Comment:
Since `alt` exists, remove it from `$el`.  (The `alt` attribute is only allowed for `img` tags)